quetz/tasks/indexing.py

```diff
@@ -210,11 +207,14 @@ def update_indexes(dao, pkgstore, channel_name, subdirs=None):
         logger.debug(f"creating indexes for subdir {sdir} of channel {channel_name}")
         raw_repodata = repo_data.export(dao, channel_name, sdir)
 
-        pm.hook.post_index_creation(
-            raw_repodata=raw_repodata,
-            channel_name=channel_name,
-            subdir=sdir,
-        )
+        try:
+            logger.debug(f"Starting post_index_creation for {sdir} of {channel_name}")
+            pm.hook.post_index_creation(
+                raw_repodata=raw_repodata, channel_name=channel_name, subdir=sdir
+            )
+            logger.debug("Finished post_index_creation for {sdir} of {channel_name}")
+        except Exception:
+            logger.exception("Exception post_index_creation:")
```

Comment:
Is there something missing? exception isn't part of the logger? or is there some magic when usign `logger.exception`?

Reply:
Ah, OK, I looked it up, and there is some magic with `logger.exception` that I wasn't aware of :)

Reply:
Yes ```logging.exception``` provides some syntactic sugar to make exception logging a bit easier.